nft_ingester/src/batch_mint/batch_mint_processor.rs

```diff
@@ -30,6 +31,17 @@ const TRANSACTION_FAIL_METRICS_LABEL: &str = "transaction_fail";
 const ARWEAVE_UPLOAD_FAIL_METRICS_LABEL: &str = "arweave_upload_fail";
 const FILE_PROCESSING_METRICS_LABEL: &str = "batch_mint_file_processing";
 
+pub async fn process_batch_mints<R: BatchMintTxSender, P: PermanentStorageClient>(
```

Comment:
The same. Cannot it be replaced with proccessor_clone.process_batch_mints in the place of invocation?